amr-wind/atmospheric_boundary_layer/convective_abl_nrel5mw/README.md

```diff
@@ -25,17 +35,144 @@ Input files are in the [input_files](input_files) directory.  There are two stag
+```
+
+### Wavenumber Spectra
+
+Two-dimensional (2D) wavenumber spectra are computed from the XY planes sampled from AMR-Wind using the [post-processing engine](https://github.com/Exawind/amr-wind-frontend/tree/main/postproengine) in the the [AMR-Wind front end](https://github.com/Exawind/amr-wind-frontend) library. The [yaml file](/postprocessing/postpro_windspectra_CGrid.yaml) for computing 2D wavenumber spectra of vertical planes sampled in this benchmark case can be called using the [ppengine.py](https://github.com/Exawind/amr-wind-frontend/blob/main/utilities/ppengine.py) utility as
+```
+python ppengine.py postpro_windspectra.yaml
+```
+Details of the 2D wavenumber computation can be found in the [documentation](edit) for the post-processing engine.
+The [ABL_wavenumber_spectra.ipynb](postprocessing/ABL_wavenumber_spectra.ipynb) Jupyter notebook plots the energy, horizontal, and vertical 2D wavenumber spectra and are reported below at the bottom-tip and hub-height locations of the NREL5MW: 
+
+![wavenumber_spectra](postprocessing/figures/ABL_wavenumber_spectra.png)
+
+### Temporal Spectra
+
+Temporal spectra are computed from the XY planes sampled from AMR-Wind using the [post-processing engine](https://github.com/Exawind/amr-wind-frontend/tree/main/postproengine) in the [AMR-Wind front end](https://github.com/Exawind/amr-wind-frontend) library. The [ABL_temporal_spectra.ipynb](postprocessing/ABL_temporal_spectra.ipynb) Jupyter notebook can be used to compute the temporal spectra and to plot the results against a Kaimal spectra. 
+Details of the temporal spectra computation can be found in the [documentation](https://github.com/Exawind/amr-wind-frontend/tree/main/postproengine/doc) for the post-processing engine. 
+
+**Note**: The path to the [AMR-Wind front end](https://github.com/Exawind/amr-wind-frontend) library must be provided in the [ABL_temporal_spectra.ipynb](postprocessing/ABL_temporal_spectra.ipynb) Jupyter notebook.  If necessary, download the library and edit the lines which define `amrwindfedirs` to include any locations of that library, e.g.,
+```python
+# Add any possible locations of amr-wind-frontend here
+amrwindfedirs = ['/projects/wind_uq/lcheung/amrwind-frontend/',
+                 '/ccs/proj/cfd162/lcheung/amrwind-frontend/']
+import sys, os, shutil, io
+for x in amrwindfedirs: sys.path.insert(1, x)
+```
+The streamwise, lateral, and vertical temporal spectra sampled from XY planes at the bottom-tip and hub-height locations of the NREL5MW are reported below:
+
+#### z = 27m
+![temporal_spectra_27](postprocessing/figures/ABL_temporal_spectra_z27.png)
+
+#### z = 90m
+![temporal_spectra_90](postprocessing/figures/ABL_temporal_spectra_z90.png)
+
+### Integral Length scale 
+
+The longitudinal and latitudinal integral length scales are computed in the [ABL_integral_lengthscale.ipynb](postprocessing/ABL_integral_lengthscale.ipynb) notebook, resulting in 
+
+- Longitudinal length scale >= 609.21 m 
```

Comment:
Just noticed one of these is >=, and the other is =.
Maybe that's right, just pointing it out.

Reply:
The lengthscales have been recomputed.